pkg/sql/sem/tree/casts_test.go

```diff
@@ -203,77 +203,3 @@ func TestTupleCastVolatility(t *testing.T) {
 		}
 	}
 }
-
-func TestCastStringToRegClassTableName(t *testing.T) {
```

Comment:
oh just noticed, where did this test go?

Reply:
pkg/sql/sem/tree/datum_test.go